qpmodel/PhysicalNode.cs

```diff
@@ -173,7 +173,21 @@ public double InclusiveCost()
                 else
                     incCost += x.InclusiveCost();
             });
-
+            //FIXED  @issue72 here should add the cost of subquery
+            if (this.logic_.filter_ != null)
```

Comment:
Subquery can show up in selection, filter and some other places - so only handle filter is not enough.

Reply:
let's first get filter case fixed and checkin other cases (selection etc) in a separate patch as it is more extensive change.

Reply:
OK. 
And I have find this SQL with uncorrealted suquery
"select c_name,(select count(O.o_orderkey) from orders O where O.o_custkey=37) as OrderCount from customer C"
has a wrong inccost.
![image](https://user-images.githubusercontent.com/37564807/93334697-f08f7c80-f857-11ea-86f0-98cf2b06eb1f.png)